tests/unittests/CMakeLists.txt

```diff
@@ -381,20 +381,18 @@ target_link_libraries(OperatorGradTest
 add_glow_test(OperatorGradTest ${GLOW_BINARY_DIR}/tests/OperatorGradTest --gtest_output=xml:OperatorGradTest.xml)
 LIST(APPEND UNOPT_TESTS ./tests/OperatorGradTest -optimize-ir=false &&)
 
-add_executable(OperatorTest
-        OperatorTest.cpp)
-target_link_libraries(OperatorTest
-                      PRIVATE
-                        BackendTestUtils
-                        Graph
-                        IR
-                        ExecutionEngine
-                        Quantization
-                        GraphOptimizer
-                        gtest
-                        TestMain)
-add_glow_test(OperatorTest ${GLOW_BINARY_DIR}/tests/OperatorTest --gtest_output=xml:OperatorTest.xml)
-LIST(APPEND UNOPT_TESTS ./tests/OperatorTest -optimize-ir=false &&)
+foreach(backend ${GLOW_BACKENDS})
```

Comment:
This should probably become a CMake macro `BUILD_TEST_FOR_ALL_BACKENDS` that can be used for different tests.